catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -322,11 +321,23 @@
     <bean id="reliableResourceDownloadManager"
           class="ddf.catalog.resource.download.ReliableResourceDownloadManager"
           init-method="init" destroy-method="cleanUp">
-        <argument ref="deprecatedProductCache"/>
-        <argument ref="retrieveStatusEventPublisher"/>
-        <argument ref="retrieveStatusEventListener"/>
-        <argument ref="downloadStatusInfo"/>
-        <argument ref="downloadThreadPool"/>
+        <cm:managed-properties persistent-id="ddf.catalog.resource.download.ReliableResourceDownloadManager"
```

Comment:
Just saw the `downloadManager.xml` file. You can ignore my comment about the "missing" `metatype.xml` file.